app/components/FileLoader.js

```diff
@@ -413,13 +415,28 @@ export default class FileLoader extends Component {
       </Table>
     );
   }
+  
+  deleteSelections = () => {
+    const store=this.props.store || {};
+	let filesToRender=_.get(store, ['fileLoadState', 'filesToRender']);
+	let i=0;
+    for(i=0;i<this.state.selections.length;i++){
+      shell.moveItemToTrash(this.state.selections[i].fullPath);
+	  filesToRender.splice(filesToRender.indexOf(this.state.selections[i]),1);
+	  _.set(store, ['fileLoadState', 'filesToRender'],filesToRender);
+    }
```

Comment:
I think it would improve readability if you used either the `for of` syntax or the `.forEach()` list operator for iterating through files. Also using `_.set()` is not the correct way to dispatch changes to the Redux store since I believe `_.set()` just modifies the object in place. Additionally, it would probably be more performant if you updated the Redux store once, after all the files have been deleted, rather than updating the store for each deleted file.